package-lock.json

```diff
@@ -4598,19 +4476,26 @@
       "dev": true
     },
     "nock": {
-      "version": "9.6.1",
-      "resolved": "https://registry.npmjs.org/nock/-/nock-9.6.1.tgz",
-      "integrity": "sha512-EDgl/WgNQ0C1BZZlASOQkQdE6tAWXJi8QQlugqzN64JJkvZ7ILijZuG24r4vCC7yOfnm6HKpne5AGExLGCeBWg==",
+      "version": "13.0.3",
```

Comment:
This looks weird.

Reply:
Not sure what happened here, but the versions match now. Hopefully that addresses the issue.